samtranslator/plugins/__init__.py

```diff
@@ -1,152 +1,10 @@
 import logging
 
-from samtranslator.model.exceptions import InvalidResourceException, InvalidDocumentException
 from enum import Enum
 
 LOG = logging.getLogger(__name__)
 
 
-class SamPlugins(object):
-    """
```

Comment:
Move this to a separate file to resolve a circular dependency.
`samtranslator.plugins (__init__.py)` -> `samtranslator.model.exceptions…` -> `samtranslator.plugins.LifeCycleEvents`